doc/geany.txt

```diff
@@ -3349,6 +3349,8 @@ Close                           Ctrl-W  (C)               Closes the current fil
 
 Reload file                     Ctrl-R  (C)               Reloads the current file.
 
+Reload all                                                Reloads all open files.
```

Comment:
Suggestion: "Reloads all open files. If changes that will be lost are detected (unsaved or saved) the reload will be confirmed, otherwise the reload will proceed without confirmation"

Reply:
Actually: "Reloads all open files. If the reload will not be 'undo'-able and changes that will be lost are detected (unsaved or saved) the reload will be confirmed, otherwise the reload will proceed without confirmation"